maps/xtehr2zibs2024/Procedure-NusringIntervention.md

```diff
@@ -281,7 +315,7 @@ The zib refers the MedicalDevice zib for "materials used", whereas the Xt-EHR mo
 | type_zib |  |
 
 ### Comments
-
+EHDSExclusive, unlikely to be used in NursingIntervention as NursingIntervention.MedicalDevice is restricted to "the materials used for the nursing procedure, such as bandages".
```

Comment:
NursingIntervention.MedicalDevice aligns with EHDSProcedure.deviceUsed. This element is unmatched.